buildenv/Build_Instructions_V8.md

```diff
@@ -386,6 +404,48 @@ bash configure --disable-ccache \
 
 :pencil: Modify the paths for freemarker and freetype if you manually downloaded and unpacked these dependencies into different directories.
 
+2) Win 32-bit
+[1] Modify the script to support the 32-bit setting
+```
+openj9-openjdk-jdk8\jdk\make\closed\autoconf\generated-configure.sh
+  if test "x$OPENJ9_CPU" = xx86-64; then
+    if test "x$OPENJDK_BUILD_OS" = xlinux; then
+      OPENJ9_PLATFORM_CODE=xa64
+    elif test "x$OPENJDK_BUILD_OS" = xwindows; then
+      OPENJ9_PLATFORM_CODE=wa64
+      if test "x$OPENJ9_LIBS_SUBDIR" = xdefault; then
+        OPENJ9_BUILDSPEC="win_x86-64"
+      if test "x$OPENJDK_TARGET_CPU_BITS" = x32; then  <----- for 32bit setting
+        OPENJ9_PLATFORM_CODE=wi32
+        OPENJ9_BUILDSPEC="win_x86"
+      fi
+    else
+        OPENJ9_BUILDSPEC="win_x86-64_cmprssptrs"
+    fi
+```
+
+[2] Specify the path to load jvm.dll for java lanncher
+```
+./jdk/make/CopyFiles.gmk
+  $(JVMCFG):
+	$(MKDIR) -p $(@D)
+	$(RM) $(@)
+        # Now check for other permutations
+        ifeq ($(JVM_VARIANT_SERVER), true)
+	  $(PRINTF) "-j9vm KNOWN\n">>$(@)  <-------------
+		$(PRINTF) "-client IGNORE\n">>$(@)
+		$(PRINTF) "-server IGNORE\n">>$(@)
+          ifeq ($(JVM_VARIANT_MINIMAL1), true)
+	    $(PRINTF) "-minimal KNOWN\n">>$(@)
+          endif
+        else
+```
```

Comment:
This change should be made at the Extensions projects.  Please open a PR for this change on 8,9,10, etc

Reply:
Note for Java 8 64-bit https://github.com/ibmruntimes/openj9-openjdk-jdk8/blob/openj9/jdk/src/solaris/bin/amd64/jvm.cfg is modified. Is https://github.com/ibmruntimes/openj9-openjdk-jdk8/blob/openj9/jdk/src/solaris/bin/i586/jvm.cfg the right place for this change to support 32-bit?

Reply:
Will do

Reply:
Is https://github.com/ibmruntimes/openj9-openjdk-jdk8/blob/openj9/jdk/src/solaris/bin/i586/jvm.cfg the right place for this change to support 32-bit?

----> I don't think it would work because jvm.cfg on 32-bit is only generated ./jdk/make/CopyFiles.gmk instead of modifying the existing jvm.cfg.

The script in ./jdk/make/CopyFiles.gmk shows:  if the target is 64-bit, then it automatically copise the jvm.cfg to the destination. That's why we don't care on win 64-bit as all setting of jvm.cfg on 64-bit are already updated by us.

Reply:
Ok. We'll want j9vm to be the only KNOWN place